lib/Interpreter/ValuePrinter.cpp

```diff
@@ -127,6 +127,7 @@ struct AccessCtrlRAII_t {
   AccessCtrlRAII_t(cling::Interpreter& Interp):
     LangOpts(const_cast<clang::LangOptions&>(Interp.getCI()->getLangOpts())) {
     savedAccessControl = LangOpts.AccessControl;
+    LangOpts.AccessControl = false;
```

Comment:
There already is, the other half of the commit exposed the bug that was obscuring the failure.